src/order/tests.py

```diff
@@ -39,35 +39,6 @@ def setUpTestData(cls):
         """
         cls.order = OrderFactory(order_item=None)
 
-    def test_order_add_item_meal_component(self):
```

Comment:
I did not remove the entire test class, as it is reused later in this test file. But, effectively, there is no more test in this class.